packages/vuetify/src/labs/VNumberInput/VNumberInput.tsx

```diff
@@ -64,11 +69,20 @@ export const VNumberInput = genericComponent<VNumberInputSlots>()({
     'update:modelValue': (val: number) => true,
   },
 
-  setup (props, { attrs, emit, slots }) {
-    const model = useProxiedModel(props, 'modelValue')
+  setup (props, { slots }) {
+    const _model = useProxiedModel(props, 'modelValue')
+
+    const model = computed({
+      get: () => _model.value,
+      set (val) {
+        if (typeof val !== 'string') _model.value = val
```

Comment:
Why aren't you using proxiedModel's transformOut argument?

Reply:
Coz I don't want it to emit anything when the value is a string, can `transform` conditionally skip an emit?

Reply:
It skips if the new value is the same as the old value, idk how reliable that is though: https://github.com/vuetifyjs/vuetify/blob/8030d407c5767f96c0474d94828b2a79b40e7c17/packages/vuetify/src/composables/proxiedModel.ts#L58-L60

Reply:
They are definitely different when typing, it emits for sure